documentation/spherex_data_access.md

```diff
@@ -17,13 +18,17 @@ Each of these data access layers is described in greater detail in the subsectio
 
 ## Browsable Directories
 
-SPHEREx on-prem data products are laid out in directories that can be navigated with standard web browsers.
+SPHEREx data products are laid out in directories that can be navigated with standard web browsers.
 This is convenient for users to get a quick sense of the types of data products that are available, to quickly download some examples by clicking through the directory tree, and to script bulk downloads using `wget` or `curl`.
 
-The root of the SPHEREx data quick release data directories is:
-https://irsa.ipac.caltech.edu/ibe/data/spherex/qr
+The root of the SPHEREx QR2 on-premises data directories is: [https://irsa.ipac.caltech.edu/ibe/data/spherex/qr2](https://irsa.ipac.caltech.edu/ibe/data/spherex/qr2).
+For QR1, replace '/qr2' with '/qr'.
+Note that QR1 is superseded by QR2 and only available through January 2026.
+All of the data products are also available on the cloud via AWS.
+Please see our [instructions for accessing cloud-hosted SPHEREx data](https://irsa.ipac.caltech.edu/cloud_access/#spherex).
 
 The public data products are organized into subdirectories based on the following organizational scheme:
+
 * **Absolute Gain Matrix:** `abs_gain_matrix/cal-agm-v[Version]-[Processing Date]/[Detector]/`
```

Comment:
I would think adding a line break would make this list more readable, right not the values are not aligned as especially that Spectral WCS have two possibility is a bit lost. This is not a blocker for this PR but something for the future?
(Or making this a definition list though that may add too much spacing)